c2corg_ui/static/partials/imageuploader.html

```diff
@@ -113,6 +113,8 @@
          ngf-keep="'distinct'"
          ngf-fix-orientation="true"
          ngf-allow-dir="true"
+         ngf-resize-if="uplCtrl.resizeIf($file, $width, $height)"
+         ngf-resize="{width: 2048, height: 2048, quality: 0.9}"
```

Comment:
Do you maybe want to move these options into a variable in the controller and use `ngf-resize="uplCtrl.resizeOptions"` here and above?